triveni-codev/GBWebApp-dev
# Darlp networks broadcast service icon 1205553547954757

Hi @triveni-codev ,

Kindly have someone check this ticket for Broadcast Service Icon Service Extension

ASANA Ticket: https://app.asana.com/0/1201943683850147/1204062341398038

To test: 
  -just add/edit a Broadcast Service Icon on Service Extension either from server/client side
  -should be able to commit and retrieve newly added/edited Broadcast Service Icon w/o any api return error  
  
Considerations: 
  -should be an ATSC3 type of network
  -should have a jpg or png file
  -this is can only be found in Service Extension (not Channel Extension)
